src/shogun/evaluation/SigmoidCalibration.h

```diff
@@ -0,0 +1,133 @@
+/*
+ * This software is distributed under BSD 3-clause license (see LICENSE file).
+ *
+ * Authors: Heiko Strathmann, Dhruv Arya
+ */
+
+#ifndef _SIGMOID_CALIBRATION_H__
+#define _SIGMOID_CALIBRATION_H__
+
+#include <shogun/lib/config.h>
+
+#include <shogun/evaluation/Calibration.h>
+#include <shogun/mathematics/Statistics.h>
+
+
+namespace shogun
+{
+	/** @brief Calibrates labels based on Platt Scaling [1]. Note that first calibration parameters need to be fitted by 
+	* calling fit_binary() or fit_multiclass(). Then call calibrate on calibrate_binary() or calibrate_multiclass() on 
```

Comment:
what will happen if a user forgets to call fit first?

Reply:
The length of the parameter vector is checked in calibrate now and an appropriate message is displayed.